integration-tests/e2e/emaildomain.manage.cy.js

```diff
@@ -0,0 +1,42 @@
+const MenuPage = require('../pages/menuPage')
```

Comment:
Are the other tests for email domain listing in a separate PR?

Reply:
Which other tests? There are separate tickets for create and delete func .

Reply:
Tests for the getAllEmailDomains.js and any others that refer to the email domain listing

Reply:
Additional test added.